package.json

```diff
@@ -21,14 +22,17 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
+    "nodemon": "^2.0.20",
```

Comment:
you dont need to use nodemon here as the tests should be informing your code rather than seeing the server running, but its not a big deal that its in here :)